### PR TITLE
Issue closeout following 0.4.0 release!

### LINKED ISSUE #10: Add test for wsgi app.
There was an issue with greenlet causing the app to except out on deploy but did not surface in testing. The app should have a small test that the `python -m app.wsgi` provides a service without any error.

### LINKED ISSUE #14: Use fluff python API rather than CLI
No reason to get subprocess in the mix here.

### LINKED ISSUE #16: Edit copy post sqlfluff org migration
Most of the copy reads like "I am a fan of sqlfluff and i like this" which is weird if the repo is now owned by the sqlfluff org.

### PR DESCRIPTION
**Pull request by nolanbconaway**
Following the release of 0.4.0 :tada:, I'll close a few issues! This PR closes:

- Fixes #10 (Add test for wsgi app.)
- Fixes #14 (Use fluff python API rather than CLI)
- Fixes #16 (Edit copy post sqlfluff org migration)

### CODE REVIEW

File: src/app/config.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"""Export global config variables."""
+import sqlfluff
+
+SQLFLUFF_VERSION = sqlfluff.__version__
+
+# manually updated on version bump
+VALID_DIALECTS = (
```
Comment (barrywhart) [inline src/app/config.py:7]: Aside: I think you could generate this automatically, if a bit hackishly, by doing something like `[os.path.basename(p) for p in os.listdir(src.sqlfluff.core.dialects)]`.

File: src/app/templates/base.html [hunk 2]
```diff
@@ -28,18 +28,23 @@
...
     <div class="container">
-        <h1>&#127881 Sqlfluff Online &#127881</h1>
+        <h1>&#127881 SQLFluff Online &#127881</h1>
 
         <p>
-            <a href='https://github.com/alanmcruickshank/sqlfluff'>Fluff</a> is a SQL formatter that I like a lot. I
-            built this app because I often want to format one-off,
-            adhoc sql that isn't even worth saving to a file. You can check out the source code for this application on
-            GitHub (<a href='https://github.com/nolanbconaway/sqlfluff-online'>link</a>).
+            <a href='https://github.com/alanmcruickshank/sqlfluff'>SQLFluff</a> is a SQL
```
Comment (barrywhart) [inline src/app/templates/base.html:34]: URL should now be: https://github.com/sqlfluff/sqlfluff

File: src/app/templates/base.html [hunk 3]
```diff
@@ -66,16 +71,14 @@ <h1>&#127881 Sqlfluff Online &#127881</h1>
 
         <div class="collapse" id=more_info>
             <p>
-                I am not the author of Sqlfluff. <a href='https://github.com/alanmcruickshank'>Alan</a> is, and many
-                other
-                contributors are (ok I opened a few PRs too). Alan never said anything like: "<i>Nolan, hack together
-                    an
-                    online tool that makes users feel bad about their code</i>". This is my
-                abomination and not his.
+                SQLFluff is maintained by a group talented and selfless people who are
```
Comment (barrywhart) [inline src/app/templates/base.html:74]: Missing word: a group [of] talented

Some people say "attractive" as well. 😄

**Review by barrywhart — CHANGES_REQUESTED** [review]

**Comment by nolanbconaway** [comment]
Fn love the attention to detail @barrywhart !

File: src/app/config.py (hunk 1 above)
Comment (nolanbconaway) [reply src/app/config.py:7]: I considered something like that, but figured that its possible not all dialects would be registered or something. Given that new dialects are fairly rare to add, I don't feel super concerned about maintaining this list.

**Comment by nolanbconaway** [comment]
I'm gonna merge this pls chastise me if there are objections

**Pull request merged**